include/circt/Dialect/Verif/VerifOps.td

```diff
@@ -200,7 +246,7 @@ def LogicEquivalenceCheckingOp : VerifOp<"lec", [
 
 def YieldOp : VerifOp<"yield", [
   Terminator,
-  ParentOneOf<["verif::LogicEquivalenceCheckingOp"]>
+  ParentOneOf<["verif::LogicEquivalenceCheckingOp", "verif::BMCOp"]>
```

Comment:
nit: Since LEC op is named of LogicEquivalenceCheckingOp, can we name BMC op as BoundedModelCheckingOp for consistency?

Reply:
Good point, thanks!